components/hab/install.sh

```diff
@@ -1,28 +1,15 @@
 #!/bin/bash
 #
-# Copyright (c) 2010-2016 Chef Software, Inc. and/or applicable contributors
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#
+set -eou pipefail
 
-# Fails on unset variables & whenever a command returns a non-zero exit code.
-set -eu
 # If the variable `$DEBUG` is set, then print the shell commands as we execute.
 if [ -n "${DEBUG:-}" ]; then set -x; fi
 
 BT_ROOT="https://api.bintray.com/content/habitat"
 BT_SEARCH="https://api.bintray.com/packages/habitat"
 
+export HAB_LICENSE="accept-no-persist"
+
```

Comment:
Any thoughts on how we can add some testing scenarios for this to our release pipeline, to ensure that everything works as desired?

Reply:
I don't have a deep understanding of our release pipeline, but if there's a way to do a fresh install into some kind of sandbox VM/container, that seems ideal.  @scotthain would likely know a lot more about this than me.  It feels a little out of scope for this particular PR, though.